.rubocop_todo.yml

```diff
@@ -59,18 +59,17 @@ Minitest/AssertRespondTo:
     - 'test/new_relic/data_container_tests.rb'
 
 # Offense count: 24
+# NOTE: As of September 2022, this cop is unsafe because it is not possible to determine whether 
+# the second argument of assert is a message or not. Leaving disabled.
```

Comment:
`Minitest/AssertWithExpectedArgument` is unsafe, so a note was added to rubocop_todo about why we should leave it. The files that had valid offenses were edited.

Reply:
I am unclear on what makes the cop "unsafe", even after reading the comment.

My hunch is that the cop is perfectly safe, but performing an autocorrection of offenses would require the `-A` flag for unsafe / may break the functionality of the code operation. Is that the case? If so, would you please update the comment to say that it's the autocorrection action that is unsafe, and not the cop itself?

Reply:
`AssertWithExpectedArgument` doesn't support autocorrect, but I called it unsafe because that's what [the docs](https://docs.rubocop.org/rubocop-minitest/cops_minitest.html#minitestassertwithexpectedargument) say about it. Maybe a better comment is:

> # NOTE: This cop detects when an 'assert' is used when it should be 'assert_equal'. As of September 2022, this cop is unable to determine whether the second argument of assert is a message or not. Disabled for now.